kiva/fonttools/font.py

```diff
@@ -98,21 +193,35 @@ class Font(object):
     def __init__(self, face_name="", size=12, family=SWISS,
                  weight=WEIGHT_NORMAL, style=NORMAL, underline=0,
                  encoding=DEFAULT):
-        if (not isinstance(face_name, str)
-                or not isinstance(size, int)
-                or not isinstance(family, int)
-                or not isinstance(weight, int)
-                or not isinstance(style, int)
-                or not isinstance(underline, int)
-                or not isinstance(encoding, int)):
+        if not isinstance(face_name, str):
+            raise RuntimeError(
```

Comment:
`RuntimeError` seems like a strange choice here. `ValueError` would seem like a better fit.

Reply:
I agree.  But `RuntimeError` is what the code previously raised in this situation.  I'll open an issue.

Reply:
Opened #933